### ISSUE TITLE
can't convert cuda:0 device type tensor to numpy

### ISSUE THREAD

**Issue by Verolelb** [lum3on/ComfyUI-StableAudioX#4]
Hi , after a war to install those nodes, i get this error at the end of the process :

![Image](https://github.com/user-attachments/assets/d5dd1735-f156-428f-aed2-d1b36d83a26d)

can you help me ?

**Comment by m-i-i-m** [comment]
The issue resides elsewhere. I ran into this as well, for anyone else stopping by. ChatGPT aiding along the way. Don't know why but here it is.
custom_nodes\comfyui-videohelpersuite\videohelpersuite\nodes.py needs to be modified. 
Line 590 & 716:
                # audio_data = audio['waveform'].squeeze(0).transpose(0,1) \
                #         .numpy().tobytes()
                audio_data = audio['waveform'].squeeze(0).transpose(0, 1).detach().cpu().numpy().tobytes()

**Comment by Verolelb** [comment]
> audio_data = audio['waveform'].squeeze(0).transpose(0, 1).detach().cpu().numpy().tobytes()

Thanks, it works like a charm

**Comment by Ragamuffin20** [comment]
> The issue resides elsewhere. I ran into this as well, for anyone else stopping by. ChatGPT aiding along the way. Don't know why but here it is. custom_nodes\comfyui-videohelpersuite\videohelpersuite\nodes.py needs to be modified. Line 590 & 716: # audio_data = audio['waveform'].squeeze(0).transpose(0,1) # .numpy().tobytes() audio_data = audio['waveform'].squeeze(0).transpose(0, 1).detach().cpu().numpy().tobytes()

i love you
